### ISSUE TITLE
Question about the initial forgetting curves

### ISSUE THREAD

**Issue by yangchenyun** [slaypni/SM-15#1]
Hi @slaypni ,
    Thanks for writing out this implementation, I have used it as a reference when [I try to implement the algorithm](https://gist.github.com/yangchenyun/4942386a91b203bea117).
    There are several initial "tweaks" to make the algorithm work properly, such as AF range, REP range etc. I found your implementation follows the description quite well. But for the forgetting curve initialization, I cannot found any relevant description about supermemo site.  The closet I could find is:

> At the beginning of the learning process, there is no repetition history and no repetition data to compute R-Factors. It will take some time before your first forgetting curves are plotted. For that reason, the initial value of the RF matrix is taken from the model of a less-than-average student. 

```
 I am curious about how you choose to setup that curve ?
```

Best,

**Comment by slaypni** [comment]
Hi @yangchenyun,
Thanks for your close look at my code. I am afraid that it could have been too complicated.

As you found, It is necessary to find out some initial curves. If I had some data collected from students like SuperMemo, I was able to utilize it. But I didn’t. So I just invented curves which may similar to “less-than-average student”. I hope it works. However, I don’t know if those are really good points or not actually.

Was I able to address your question? I wish you can complete your implementation.

**Comment by yangchenyun** [comment]
Hi @slaypni , 
    After reading the algorithm, it is not hard to follow your code; and it helps me understand further about the original algorithm and to implement my own, thanks for open sourcing it! 
    When you are writing out the initial curves, do you read some references or you estimate the "less-than-average-student" by yourself? I am curious about that choice you made behind the scene.

**Comment by slaypni** [comment]
The way I wrote the initial curves was not sophisticated at all. First, I assumed that it could be represented by exponential function because forgetting curves are modeled by exponential regression in SuperMemo. Then I tried to find how to set coefficients and parameters _a_ and _r_ by modifying them  many times watching the resulting shape of the function using WolframAlpha. The criteria to see how it is good was the images of forgetting curves on the SuperMemo website. I tried to imitate the shape. In addition to that, The forgetting curve in my brain was also another standard to some degree.
